tests/e2e/default/tasks/tasks-breadcrumbs.wdio-spec.js

```diff
@@ -23,27 +21,24 @@ describe('Tasks tab breadcrumbs', () => {
     type: 'health_center',
     parent: { _id: districtHospital._id },
   });
+
   const chwContact = {
```

Comment:
Because for some reason if a use the `personFactory` for those two (`chwContact` and `supervisorContact`) the tasks are in a different order and one more task is created, and I didn't want to change the expected values.

This is how it looks with the objects as they are:
![image](https://github.com/user-attachments/assets/d2bed69b-41d1-48af-a19a-8ddf5199cd8e)

And this is how it looks if I use factories:

```.js
  const chwContact = personFactory.build({
    _id: 'fixture:user:user1',
    name: 'chw',
    phone: '+12068881234',
    type: 'person',
    place: healthCenter1._id,
    parent: healthCenter1
  });
  const supervisorContact = personFactory.build({
    _id: 'fixture:user:user2',
    name: 'supervisor',
    phone: '+12068881235',
    type: 'person',
    place: districtHospital._id,
    parent: districtHospital
  });
```

![image](https://github.com/user-attachments/assets/f4cf2c71-1d65-4115-8a29-d132a1d6cb9a)

What do you think @lorerod ? should I change the expected values in the assert?

Reply:
I can only think of is that the person factory sets `patiend_id` to a value, but why not generate tasks also for the supervisor? The parent place may have something to do with it. Do you mind setting `patien_id` to a null value to try?

Reply:
Great question, I don't know what is different either between the `cht` and the `supervisor`, but adding the null value had the same result
![image](https://github.com/user-attachments/assets/d6a33711-242b-46c3-94d8-0d5383aedd04)

Reply:
Ok, I understand now

the important attribute was the `reported_date`, in the config file, [here](https://github.com/medic/cht-core/blob/master/tests/e2e/default/tasks/config/tasks-breadcrumbs-config.js#L8) is where it evaluates the date. So if I set the `reported_date` as null it will work.

And the creation of the `chw` or `supervisor` was due to the user that was logged in.

Thank you Lore! ⭐